server.js

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
We already have `Controller.js` for routing pages so we don't need this file.

Reply:
I didn't find any solution apart from creating a express dummy server to solve the Cross-origin resource sharing (CORS) issue. It is a mechanism that allows restricted resources on a web page to be requested from another domain outside the domain from which the first resource was served.. Because Instagram API to work from client side(Browser) to need to pass it through the server. If you have any solution for this issue please share with me i would love to implement it.